MatrixSDK/MXDefaultBackgroundModeHandler.h

```diff
@@ -0,0 +1,14 @@
+//
```

Comment:
Could you please use the Apache header?
You can use your name in like https://github.com/matrix-org/matrix-ios-sdk/blob/master/MatrixSDK/SwiftMatrixSDK.h